dotCMS/src/main/java/com/dotmarketing/startup/runonce/Task04305UpdateWorkflowActionTable.java

```diff
@@ -60,10 +64,15 @@
     private static final String MSSQL_ADD_SCHEME_ID_COLUMN    = "ALTER TABLE workflow_action ADD scheme_id NVARCHAR(36)";
     private static final String ORACLE_ADD_SCHEME_ID_COLUMN   = "ALTER TABLE workflow_action ADD scheme_id varchar2(36)";
 
-    private static final String MYSQL_ADD_REQUIRES_CHECKOUT_OPTION_COLUMN    = "ALTER TABLE workflow_action ADD requires_checkout_option VARCHAR(16)  default 'both'";
-    private static final String POSTGRES_ADD_REQUIRES_CHECKOUT_OPTION_COLUMN = "ALTER TABLE workflow_action ADD requires_checkout_option VARCHAR(16)  default 'both'";
-    private static final String MSSQL_ADD_REQUIRES_CHECKOUT_OPTION_COLUMN    = "ALTER TABLE workflow_action ADD requires_checkout_option NVARCHAR(16) default 'both'";
-    private static final String ORACLE_ADD_REQUIRES_CHECKOUT_OPTION_COLUMN   = "ALTER TABLE workflow_action ADD requires_checkout_option varchar2(16) default 'both'";
+    private static final String MYSQL_NOT_NULL_CONSTRAINT_SHOW_ON_COLUMN     = "ALTER TABLE workflow_action MODIFY       scheme_id VARCHAR(36)  NOT NULL;";
+    private static final String POSTGRES_NOT_NULL_CONSTRAINT_SHOW_ON_COLUMN  = "ALTER TABLE workflow_action ALTER COLUMN scheme_id SET          NOT NULL;";
+    private static final String MSSQL_NOT_NULL_CONSTRAINT_SHOW_ON_COLUMN     = "ALTER TABLE workflow_action ALTER COLUMN scheme_id NVARCHAR(36) NOT NULL";
+    private static final String ORACLE_NOT_NULL_CONSTRAINT_SHOW_ON_COLUMN    = "ALTER TABLE workflow_action MODIFY       scheme_id varchar2(36) NOT NULL";
+
+    private static final String MYSQL_ADD_REQUIRES_CHECKOUT_OPTION_COLUMN    = "ALTER TABLE workflow_action ADD show_on varchar(255)  default 'LOCKED,UNLOCKED'";
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Split this 161 characters long line (which is greater than 150 authorized). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00103)